src/modules/stake-dot/components/Balance/Balance.tsx

```diff
@@ -0,0 +1,68 @@
+import { Paper, Typography, Button, Box, Tooltip } from '@material-ui/core';
+import React from 'react';
+import BigNumber from 'bignumber.js';
+
+import { t } from '../../../../common/utils/intl';
+import { ReactComponent as DOTIcon } from '../../assets/DOT.svg';
+
+import { useBalanceStyles } from './BalanceStyles';
+import { Body2 } from '../../../../UiKit/Typography';
+
+interface IBalanceProps {
+  amount: BigNumber;
+  isConnected?: boolean;
+}
+
+export const Balance = ({ amount, isConnected = false }: IBalanceProps) => {
+  const classes = useBalanceStyles();
+
+  return (
+    <Paper variant="outlined" square={false} className={classes.root}>
+      <div className={classes.header}>
+        <Typography variant="body1" className={classes.headerContent}>
+          <DOTIcon />
+          <div className={classes.headerText}>
+            {t('stake-dot.dashboard.aDotb-balance')}
+          </div>
+        </Typography>
+        <Body2>
+          <a
+            href="https://polkadot.network/"
+            target="_blank"
+            rel="noreferrer"
+            className={classes.whatIs}
+          >
+            {t('stake-dot.dashboard.what-is')}
+          </a>
+        </Body2>
+      </div>
+
+      <div className={classes.footer}>
+        {isConnected ? (
+          <div className={classes.amount}>
+            <Typography variant="h2">{amount.toFixed(2)}</Typography>
```

Comment:
```suggestion
            <Headline2>{amount.toFixed(2)}</Headline2>
```
The same question as above

Reply:
to make `<Typography variant="h2">` and `<Typography variant="h6">` equally it's better to use` <Typography>` - there is no `Headline6`

Reply:
Why do you need to make it equal?

Reply:
it's easier to see the difference when the code is written the same